department-of-veterans-affairs/abd-vro
# MCP-2037 Metrics Page Improvements

## What was the problem?
We want to include more count totals in the metrics service.  These counts include total assessment results, medications, total blood pressure readings, recent blood pressure readings, and procedures.

Associated tickets or Slack threads:

- [MCP-2037](https://amida.atlassian.net/browse/MCP-2037)

## How does this fix it?[^1]

Includes assessmentResult, medicationsCount, totalBpReadings, recentBpReadings, and proceduresCount to the claim-metrics endpoint

## How to test this PR
Step 1: populate db with claims and assessment results by executing the full health assessment endpoint
Step 2: execute claim-metrics endpoint
Step 3: Verify the total claims, assessmentResults, and EvidenceSummaryDocuments are the right counts.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
